hosts/azure/jenkins-controller/configuration.nix

```diff
@@ -106,6 +106,7 @@ in
         zstd
         jq
         csvkit
+        curl
```

Comment:
What is this used for? Trying to figure out if it belongs here...

Reply:
Curl is used to digt out Jenkins infop when emailing possible bild breakers